create-packages.sh

```diff
@@ -41,6 +41,11 @@ awk '{gsub(/cartridge.cfg\({/, "&\n    vshard_groups = { hot = { bucket_count =
     ${appname}/init.lua >${appname}/temp.lua
 mv ${appname}/temp.lua ${appname}/init.lua
 
+awk '{gsub(/-- dependencies/, "dependencies")}1' \
```

Comment:
Is it was in all versions of default app? What about to add dependencies, not uncomment?

Reply:
Yes, it is.
I think that it's enough now - this code doesn't seem to be changed.